server/schemas/schemaModel.js

```diff
@@ -1,10 +1,53 @@
 var mongoose = require("mongoose");
-
+var bluebird = require("bluebird");
+var helpers = require("../config/helpers");
 var Schema = mongoose.Schema;
-
 var schemaSchema = new Schema({
```

Comment:
We probably need to figure out a better name than 'schema' for validation. schemaSchema is confusing.